.taskcluster.yml

```diff
@@ -0,0 +1,50 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+version: 0
+allowPullRequests: public
+tasks:
+###############################################################################
+# Task: Pull requests and pushes
+#
+# Triggered whenever a pull request is opened or updated, or a push is being made
+#
+# - Build the library
+# - Run unit tests
+# - Run code quality tools
+###############################################################################
+  - provisionerId: '{{ taskcluster.docker.provisionerId }}'
+    workerType: '{{ taskcluster.docker.workerType }}'
+    extra:
+      github:
+        events:
+          - pull_request.opened
+          - pull_request.edited
+          - pull_request.synchronize
+          - pull_request.reopened
+          - push
+    payload:
+      maxRunTime: 3600
+      deadline: "{{ '2 hours' | $fromNow }}"
+      image: 'mozillamobile/fretboard'
```

Comment:
This is not the name of the repository - but the name of a Docker image:
https://docs.docker.com/

Those images are build from a `Dockerfile`, e.g.:
https://github.com/mozilla-mobile/android-components/blob/master/automation/docker/Dockerfile

After the image is created it is uploaded to Docker Hub where taskcluster can find it and pull it from.

You could reuse the android-components image for now (mozillamobile/android-components:1.1). You'll just need to checkout the right repository. See below.